docs/en/operations/settings/settings.md

```diff
@@ -211,6 +211,11 @@ Possible values:
 
 Default value: 0.
 
+## input_format_null_as_default {#settings-input_format_null_as_default}
```

Comment:
Shouldn't it be `input_format_csv_null_as_default`?

Suppose we want to implement similar logic to `TSV`. But in TSV it's much more dangerous, because there is no way to unambiguate from NULL and a String with 'NULL' value. But still usable (e.g. user has only numeric fields in a table). And this gives the motivation to allow the user to enable this logic separately for CSV and TSV.

Reply:
This setting is about replacing nulls with default values for non-nullable data types (#2633 and #6033), not about parsing `NULL` as `\N` (`input_format_null_as_default` and `input_format_csv_unquoted_null_literal_as_null` are independent). Should we add a separate setting `*_null_as_default` for each format?

Reply:
No. It's fine :)